Source/Creation/Frame/Content/ID3CommentFrameCreator.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ID3CommentFrameCreator.swift
+//
+//  Created by Fabrizio Duroni on 26/02/2018.
+//  2018 Fabrizio Duroni.
+//
+
+import Foundation
+
+class ID3CommentFrameCreator: ID3StringFrameCreator {
+    override func createFrames(id3Tag: ID3Tag, tag: [UInt8]) -> [UInt8] {
+        if let commentFrame = id3Tag.frames[.Comment] as? ID3FrameWithStringContent {
+            return createFrameUsing(frameType: .Comment, content: commentFrame.content, id3Tag: id3Tag, andAddItTo: tag)
```

Comment:
As from the id3.org specification this frame should contain some other informations like:
- Language               $xx xx xx
- Short content descrip. <text string according to encoding> $00 (00)

See http://id3.org/d3v2.3.0 section 4.11

Reply:
I'm not entirely sure if I'll be able to add that at my current skill level. I may omit it if I can't.

The reason I included it is because I'm working on a project that involves tagging audiobooks, and the comment frame is the preferred "short description" location for audiobooks, with the unsynchronized lyrics frame being the preferred "long description" location. It does work in practice as-is, so if I can't implement the change you've requested here, I'll likely remove it from the official pull request and just use it in an unofficial version intended for my own project.

Reply:
I've gone ahead and removed this frame entirely. And language as well, since they both require a format that I can't implement just by replicating the frames you've provided.

Reply:
Thank you. In this library I try to be

Reply:
ps. remeber to push the modification because I don't see the new commit